x-pack/plugins/dataset_quality/server/routes/types.ts

```diff
@@ -18,6 +18,7 @@ export interface DatasetQualityRouteHandlerResources {
   context: DatasetQualityRequestHandlerContext;
   logger: Logger;
   request: KibanaRequest;
+  response: KibanaResponseFactory;
```

Comment:
Do we really need this? We could just throw using `Boom` and avoid to have this property here